src/components/SimpleTable.js

```diff
@@ -106,6 +108,7 @@ export default function SimpleTable(props) {
     rowData = [], //rowData -> {data, component, style, onClick}
     isLoading,
     styles, //styles -> baseStyle, columnStyle needed to determine sizing and color assignments
+    rowWrapper = ({renderedRow}) => renderedRow, // ({rowData, renderedRow}) => ... : allows injecting custom container to row
```

Comment:
One thing that's jarring is the styling of the loading phase before the actual content is rendered. Not sure how that's defined, but I think we need to make that more of a spinner than a giant table that goes away within a second.

Reply:
I changed the table loading animation to be a spinner in the middle of the screen, and the table is simply empty until the data is fetched. This affects all tables, but I think it looks pretty good, even for the larger tables. If preferred, we could make it a flagged prop if you want the giant table loading animation on the, e.g., admin user page

Reply:
This looks good, thanks for the change!